packages/core/src/components/private/FormInput.tsx

```diff
@@ -26,12 +29,14 @@ export type IgnoreAttributes =
   | 'security';
 
 export type Props<T = any> = {
-  /** Decrease font size and padding. */
+  /** Decrease font size and padding to small. */
   compact?: boolean;
```

Comment:
Add a new `small` prop, and if `compact` is used, set it to small and log a deprecation warning in dev.

Reply:
Can you add a `@deprecated` to the beginning of this comment.